compiler/executable/pipeline.rs

```diff
@@ -91,38 +91,17 @@ pub fn compile_pipeline(
     annotated_fetch: Option<AnnotatedFetch>,
     input_variables: &HashSet<Variable>,
 ) -> Result<ExecutablePipeline, ExecutableCompilationError> {
-    // TODO: Cache compiled schema functions?
-    let mut executable_schema_functions = HashMap::new();
-    let schema_tabling_requirements = determine_tabling_requirements(
-        &annotated_schema_functions.iter().map(|(id, function)| (FunctionID::Schema(id.clone()), function)).collect(),
-    );
-    for (id, function) in annotated_schema_functions.iter() {
-        // TODO: We could save cloning the whole function and only clone the stages.
-        let compiled = compile_function(
-            statistics,
-            &ExecutableFunctionRegistry::empty(),
-            function.clone(),
-            schema_tabling_requirements[&FunctionID::Schema(id.clone())],
-        )?;
-        executable_schema_functions.insert(id.clone(), compiled);
-    }
-    let arced_executable_schema_functions = Arc::new(executable_schema_functions);
+    // TODO: Cache compiled schema functions so we don't have to clone here.
```

Comment:
Update comment: `TODO: we could cache compiled schema functions so we dont have to re-compile with every query here`